packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuPanel.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { AccordionPanel, AccordionPanelProps } from '../Accordion';
+import { DropdownExpandableMenuGroup } from './DropdownExpandableMenuGroup';
+
+export interface DropdownExpandableMenuPanelProps extends AccordionPanelProps {}
+
+const StyledAccordionPanel = styled(AccordionPanel)`
+  padding: 0;
+`;
+
+export const DropdownExpandableMenuPanel = React.forwardRef<
```

Comment:
If we are forwarding the ref, then we need to use it (otherwise we get console warnings).